front/src/components/RegistrationUser/RegistrationUser.component.html

```diff
@@ -0,0 +1,73 @@
+  <div>   
+    <el-button type="primary" @click="VisibleRegistrationForm(true)">Зарегистрироваться</el-button>
+    <el-dialog 
+    title="Создать аккаунт"
+    :visible.sync="isVisibleRegistrationForm" 
+    center
+    :show-close="false"
+    width= "464px"
+    >
+      <div class="close"><el-button icon="el-icon-close" circle @click="VisibleRegistrationForm(false)"></el-button> </div> 
+      <el-form :model="ruleForm"
+      :rules="rules"
+      >              
+        <el-form-item class="item" prop="userName">
+        <el-input placeholder="Имя пользователя"
+                  v-model="ruleForm.userName"
+        >
+        </el-input>
+        </el-form-item>
+        <el-form-item class="item" prop="email">
+        <el-input placeholder="Email"
+                  @input="updateMailBusy()"
+                  v-model="ruleForm.email"
+        >
+        </el-input>
+        <el-alert v-show="MailboxBusy"
+                  title="Данный e-mail уже зарегистрирован"
+                  type="error"
+                  :closable="false"
+                  show-icon
+        >
+        </el-alert>
+        </el-form-item>
+        <el-form-item class="item" prop="password">
+        <el-input type="password"
+                  placeholder="Пароль"
+                  v-model="ruleForm.password"
+        >
+        </el-input>
+        </el-form-item>
+      </el-form>
+      
+      <span slot="footer" class="dialog-footer">
+        <el-button
+            id='Registration'
+            type="primary"
+            @click="Registration()"
```

Comment:
register - not registration, read article about naming in wiki https://github.com/horba/film-finity/wiki/JS-Code-Style-Guide

register() - in lowerCase read our guide https://github.com/horba/film-finity/wiki/Frontend-Namings-and-Structure